addon/components/hyper-table-v2/filtering-renderers/date.hbs

```diff
@@ -25,7 +25,7 @@
           <EmberFlatpickr @onChange={{this.selectFixedDate}} @dateFormat="M/D/Y"
                           @placeholder={{t "hypertable.column.filtering.date.range.label"}}
                           @mode="range" @date={{readonly this._currentDateValue}} @altInput={{true}}
-                          @altInputClass="upf-input upf-input--textarea"
+                          @altInputClass="upf-input"
```

Comment:
took the occasion to cleanup these `upf-input--textarea` (you'll find multiple of these in the PR)